README.md

```diff
@@ -259,6 +259,15 @@ Default: `['passwd', 'password', 'secret', 'confirm_password', 'password_confirm
 Default: `[]`
 </dd>
 
+<dt>showUnhandledErrors
+</dt>
+<dd>Whether or not to show unhandled errors on the console before sending them to Rollbar.  By default, unhandled errors will be consumed and silently sent to Rollbar. 
+
+Default: `false`
```

Comment:
When @coryvirok and I discussed this last week, he suggested a default of `false` to keep behavior the same.  Easy to change, I have no personal preference.